functions/src/import-employees-from-alibeez.ts

```diff
@@ -47,11 +48,9 @@ export const importEmployeesFromAlibeez = async (config: AlibeezConfig) => {
     return;
   }
   const employees: AlibeezEmployee[] = await response.json();
-  const employeesWithValidEmail = employees.filter(
-    employee =>
-      employee.zenikaEmail && employee.zenikaEmail.endsWith("@zenika.com")
-  );
-  const importRef = firebase
+  const[ employeesWithValidEmail, employeesWithNoValidEmail ] = partition(employees,function (o) { return o.zenikaEmail && o.zenikaEmail.endsWith('@zenika.com') })
```

Comment:
This code looks like it is not formatted like the rest of the code. Though it is not obvious, this project uses prettier to format code. Install the prettier extension for your editor and format this file to get the conforming formatting.

Additional little details:
- I invite you to use an arrow function instead of a full function as the callback to `partition`
- I also invite you to come up with a better name for the variable `o`